src/python/turicreate/toolkits/activity_classifier/_tf_model_architecture.py

```diff
@@ -18,9 +18,9 @@
 CONV_H = 64
 LSTM_H = 200
 DENSE_H = 128
```

Comment:
Please revert the changes on this file since you haven't actually touched anything.